apiserver/Makefile

```diff
@@ -232,7 +227,7 @@ run-kubernetes-server: config/crd run-etcd stop-kubernetes-server
 		-v $(CURDIR)/test/certs:/home/user/certs \
 		lachlanevenson/k8s-kubectl:${KUBECTL_VERSION} \
 		--kubeconfig=/home/user/certs/kubeconfig \
-		apply -f /manifests/config/crd/
+		apply -f /go/src/github.com/projectcalico/calico/libcalico-go/config/crd/
```

Comment:
this doesn't seem to pick up the correct path, seems like `:231` needs to change from the current `-v  $(CURDIR):/manifests`